src/services/car/messageLead.ts

```diff
@@ -55,3 +62,31 @@ export const sendMessageLead = async ({
     })
   }
 }
+
+export const fetchDealerMessageLeads = async ({
+  dealerId,
+  page = 0,
+  size = 7,
```

Comment:
We also expose default pagination for listings search so we don't need to declare that in the project. Can we do that too?

Reply:
Were are the defined default pagination values for listings. I could not find them.

Reply:
In the listing search https://github.com/carforyou/carforyou-api-client-pkg/blob/master/src/services/search/listingSearch.ts#L99
We also have helpers to provide defaults here: https://github.com/carforyou/carforyou-api-client-pkg/blob/b6615a2b5a3f1436e350daeb8c5cedf01c02dcaf/src/lib/pageParams.ts#L1